package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should I delete this too?

Reply:
Yeah, let’s keep the js dev stuff fully encapsulated to frontend/